mkosi/__init__.py

```diff
@@ -1442,6 +1442,38 @@ def want_signed_pcrs(config: Config) -> bool:
     )
 
 
+def add_ukify_secure_boot_arguments(
+    context: Context, out_cmd: list[PathString], out_options: list[PathString]
+) -> None:
```

Comment:
Mutating input arguments makes things harder to reason about. I'd prefer this to return the extra cmd argument and options and adding them where they are used.